dojo/models.py

```diff
@@ -1413,101 +1413,288 @@ class Finding(models.Model):
-    sast_source_line = models.IntegerField(null=True, blank=True,
+    sourcefilepath = models.TextField(null=True,
+                                      blank=True,
+                                      editable=False,
+                                      verbose_name="Source File Path",
+                                      help_text="Filepath of the source code file in which the flaw is located.")  # Not used? to remove
+    sourcefile = models.TextField(null=True,
+                                  blank=True,
+                                  editable=False,
+                                  verbose_name="Source File",
+                                  help_text="Name of the source code file in which the flaw is located.")
+    param = models.TextField(null=True,
+                             blank=True,
+                             editable=False,
+                             verbose_name="Parameter",
+                             help_text="Parameter used to trigger the issue (DAST).")
+    payload = models.TextField(null=True,
+                               blank=True,
+                               editable=False,
+                               verbose_name="Payload",
+                               help_text="Payload used to attack the service / application and trigger the bug / problem.")
+    hash_code = models.CharField(null=True,
+                                 blank=True,
+                                 editable=False,
+                                 max_length=64,
+                                 verbose_name="Hash Code",
+                                 help_text="A hash over a configureable set of fields that is used for findings deduplication.")
+    line = models.IntegerField(null=True,
+                               blank=True,
                                verbose_name="Line number",
-                               help_text="Source line number of the attack vector")
-    sast_source_file_path = models.CharField(null=True, blank=True, max_length=4000, help_text="Source filepath of the attack vector")
-    nb_occurences = models.IntegerField(null=True, blank=True,
-                               verbose_name="Number of occurences",
-                               help_text="Number of occurences in the source tool when several vulnerabilites were found and aggregated by the scanner")
+                               help_text="Line number. For SAST, when source (start of the attack vector) and sink (end of the attack vector) information are available, put sink information here.")
+    file_path = models.CharField(null=True,
+                                 blank=True,
+                                 max_length=4000,
+                                 verbose_name="File path",
+                                 help_text="File name with path. For SAST, when source (start of the attack vector) and sink (end of the attack vector) information are available, put sink information here.")
```

Comment:
This one as well

Reply:
A real like example with Anchore would be `/usr/share/webapp/webapp.war:WEB-INF/plugins/some-plugins.hpi:WEB-INF/lib/jackson-databind-2.7.3.jar`

In the UI, the header is "Location".

Maybe something simpler like "Identified file(s) containing the vulnerability"?

Reply:
+1 on the "identified files" idea

Reply:
Good idea @madchap, I've put flaw instead of vulnerability as some issues are not necessarily vulnerabilities and it's the word used in many places in the doc.